portfolio/src/main/webapp/index.html

```diff
@@ -116,6 +116,9 @@ <h1 class="title"> Leave a comment! </h1>
     <br>
     <br>
     <br>
+    <div>
+    <h2 class="comment-header"> Comments: </h2>
```

Comment:
nit: add indentation on this <h2> element, so it will be 1 level deeper inside the <div> :D